typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -33,7 +32,8 @@ import inevmEthereumUsdtAddresses from './warp/inevm-USDT-addresses.json';
 import injectiveInevmInjAddresses from './warp/injective-inevm-addresses.json';
 import mantaTIAAddresses from './warp/manta-TIA-addresses.json';
 import merklyEthAddresses from './warp/merkly-eth-addresses.json';
-import renzoEzEthAddresses from './warp/renzo-ezETH-addresses.json';
+import renzoEzEthAddressesV1 from './warp/renzo-ezETH-addresses-v1.json';
```

Comment:
is this v2?

Reply:
Hmm I guess you are right, v2 introduced fraxtal

Reply:
contract addresses wont change